exporting/init_connectors.c

```diff
@@ -64,7 +64,8 @@ int init_connectors(struct engine *engine)
             }
 
             // dispatch the instance worker thread
-            uv_thread_create(&instance->thread, connector->worker, instance);
+            if (uv_thread_create(&instance->thread, connector->worker, instance))
```

Comment:
A suggestion, since the version 1.4.1(http://docs.libuv.org/en/v1.x/threading.html#c.uv_thread_create)  `uv_thread_create` returns UV_E* errors, so we could use `uv_strerror`  to give more details about the error that happened when we were trying to create a thread, could you add a call for `uv_strerror` for we store the information inside `error.log`?

Reply:
Done.

Reply:
Thank you! :)